home/server-installation.html

```diff
@@ -63,11 +64,12 @@ <h2>3.b. Installation on Linux</h2>
 <h3>Using our build (recommended)</h3>
 <p>This step will work on Debian, Ubuntu, and similar Debian-derivatives on x86_64. If you're on a different distribution or architecture, refer to the "Building from source” step below.</p>
```

Comment:
> This step will work on all distributions we provide binaries for here

and then link to the github releases there

Reply:
As in, id like this to also not be distro specific at all

Reply:
Corrected that.
The same could be said for [this](https://github.com/BeamMP/Wiki/pull/2/files#diff-f984a1d8290094db9a470d316aa8f0e8a6dcad09e28b02589e0cbd3b4066e057R78) as well, couldn't it?
But I can't write there something like this:
> BeamMP-Server for Linux can be officially built on all distributions that already have a binary [here].

As Arch does not have a binary but can you can still officially build the server on it.
Since you want to have the wiki less distro-specific how do I proceed?
Do I make a small table/list of the distros where the server can be officially built on or would something like this work as well?
> Other distributions in addition to the ones that already have a binary [here] are likely to work, too, but aren't officially supported.

Reply:
I think the wording that is there is sufficient.

Reply:
We cannot be distro-specific anywhere, because we can (with the new build system, and with the rust server as well) build on pretty much any distro with very little work. I may add an alpine build for fun during a work break, and then all docs would need to be updated because that's not a debian derivative. So yeah, please just keep it as generic as possible.

We do *now* support all the platforms that we build for, so all distros the latest release is built for are supported. This may change every now and then, so wording like that is best IMO <3

Reply:
So the wording everywhere this would be mentioned should be 

> all distributions for which binaries exist [here]

with [here] linking to the latest release (https://github.com/BeamMP/BeamMP-Server/releases/latest) which automatically forwards you to the latest release.

Reply:
Done both here and in the Docs.